fastai/vision/learner.py

```diff
@@ -154,6 +154,7 @@ def _add_norm(dls, meta, pretrained):
     if not pretrained: return
     stats = meta.get('stats')
     if stats is None: return
+    if dls.after_batch.fs.filter(lambda tfm: isinstance(tfm, Normalize)): return
     dls.add_tfms([Normalize.from_stats(*stats)],'after_batch')
```

Comment:
if you change the previous line to `not` then you could move this to the previous line, and get rid of the `return` there